### ISSUE TITLE
SMTP State Documentation missing note on when the attachements were added

### ISSUE THREAD

**Issue by claudekenni** [saltstack/salt#52958]
### Description of Issue/Question
The smtp state module documentation has the attachements as a keyword, but this module does not have the attachments keyword on my 2018.3.0 SLES Minion

```
cat /usr/lib/python2.7/site-packages/salt/states/smtp.py | grep 'def send_msg'
def send_msg(name, recipient, subject, sender=None, profile=None, use_ssl='True'):
```

### Versions Report
(Provided by running `salt --versions-report`. Please also mention any differences in master/minion versions.)
```
salt-call --versions-report
Salt Version:
           Salt: 2018.3.0

Dependency Versions:
           cffi: 1.1.0
       cherrypy: Not Installed
       dateutil: Not Installed
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: 2.14
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 2.7.13 (default, Jan 11 2017, 10:56:06) [GCC]
   python-gnupg: Not Installed
         PyYAML: 3.12
          PyZMQ: 15.2.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.4.2
            ZMQ: 4.2.1

System Versions:
           dist: SuSE 12 x86_64
         locale: UTF-8
        machine: x86_64
        release: 3.12.74-60.64.40-default
         system: Linux
        version: SUSE Linux Enterprise Server  12 x86_64

```

**Comment by claudekenni** [comment]
File in current DEV

https://github.com/saltstack/salt/blob/2aa183284b9e85011706474c9cd65cf6d2125e13/salt/states/smtp.py#L30

**Comment by garethgreenaway** [comment]
@claudekenni Thanks for the report. The `attachments` parameter is available beginning in the 2019.2 release, https://docs.saltstack.com/en/latest/ref/states/all/salt.states.smtp.html.  The Salt documentation website defaults to the latest version of the stable branch, which is 2019.2.0.  The other tabs available are the current released version of the previous stable branch, in this case 2018.3.4,  and the develop branch.

**Comment by claudekenni** [comment]
Thanks for clarification, this should work for me.
